src/main/java/io/vertx/redis/RedisClient.java

```diff
@@ -1833,7 +1834,20 @@ static RedisClient create(Vertx vertx, RedisOptions config) {
    * group: string
    */
   @Fluent
-  RedisClient setBinary(String key, String value, Handler<AsyncResult<Void>> handler);
+  RedisClient setBinary(String key, Buffer value, Handler<AsyncResult<Void>> handler);
+
+  /**
+   * Set the string value of a key
+   *
+   * @param key     Key of which value to set
+   * @param value   New value for the key
+   * @param options Set options
+   * @param handler Handler for the result of this call.
+   * @since 1.0.0
```

Comment:
that is the redis server api level all methods are annotated with the redis api server that supports it, it was something that was copied from the original client that i wrote. I can remove them all since the javadoc annotation does not refer to the API version